<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="user_menu">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="exitLogin_action"/>
    <addaction name="changePD_action"/>
   </widget>
   <addaction name="user_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actions">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="actionxue">
   <property name="text">
    <string>xue</string>
   </property>
  </action>
  <action name="actions_2">
   <property name="text">
    <string>个人信息</string>
   </property>
  </action>
  <action name="exitLogin_action">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
  <action name="changePD_action">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
